### ISSUE TITLE
[REVIEW]: akutils: Facilitating analyses of microbial communities through QIIME.

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#57]
**Submitting author:** @alk224 (<a href="http://orcid.org/0000-0002-7057-9957">Andrew Krohn</a>)
**Repository:** <a href="https://github.com/alk224/akutils-v1.2" target ="_blank">https://github.com/alk224/akutils-v1.2</a>
**Version:** v1.2.3
**Editor:** @arfon 
**Reviewer:** @serine 
## Status

[![status](http://joss.theoj.org/papers/00608bdd0dcee9c442427f3d8c1c5def/status.svg)](http://joss.theoj.org/papers/00608bdd0dcee9c442427f3d8c1c5def)

Status badge code:

```
HTML: <a href="http://joss.theoj.org/papers/00608bdd0dcee9c442427f3d8c1c5def"><img src="http://joss.theoj.org/papers/00608bdd0dcee9c442427f3d8c1c5def/status.svg"></a>
Markdown: [![status](http://joss.theoj.org/papers/00608bdd0dcee9c442427f3d8c1c5def/status.svg)](http://joss.theoj.org/papers/00608bdd0dcee9c442427f3d8c1c5def)
```

**Reviewers and authors**: Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue
in the <a href="https://github.com/alk224/akutils-v1.2" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers)
in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice
versa will create corresponding breadcrumb trails in the link target.)
### Conflict of interest
- [x] As the reviewer I confirm that there are no conflicts of interest for me to review this work (such as being a major contributor to the software).
### General checks
- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/alk224/akutils-v1.2">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Version:** Does the release version given match the GitHub release (v1.2.3)?
### Functionality
- [ ] **Installation:** Does installation proceed as outlined in the documentation?
- [ ] **Functionality:** Have the functional claims of the software been confirmed?
- [ ] **Performance:** Have any performance claims of the software been confirmed?
### Documentation
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g. API method documentation)?
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support
### Software paper

Paper PDF: [10.21105.joss.00057.pdf](https://github.com/openjournals/joss-reviews/files/481137/10.21105.joss.00057.pdf)
- [x] **Authors:** Does the `paper.md` file include a list of authors with their affiliations?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **References:** Do all archival references that should have a DOI list one (e.g. papers, datasets, software)?

**Comment by arfon** [comment]
Hi @alk224 - thanks for submitting this. Before we can proceed with this submission please take a look at the author guidelines (http://joss.theoj.org/about#author_guidelines) on how to format the `paper.md` file. Specifically, the metadata needs to be formatted _exactly_ as the example shows below. In addition, the references need to be put in a separate bibtex file named `paper.bib` and citations to these entries made using [this formatting](http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citations).

```
---
title: 'Fidgit: An ungodly union of GitHub and figshare'
tags:
  - example
  - tags
  - for the paper
authors:
 - name: Arfon M Smith
   orcid: 0000-0002-3957-2474
   affiliation: GitHub Inc.
date: 14 February 2016
bibliography: paper.bib
---

# Summary

Your summary here....

# References
```

**Comment by alk224** [comment]
Thanks Arfon,

Will try to make the necessary changes this weekend. I appreciate the help.


--
Andrew Krohn
PhD Candidate, NAU Biological Sciences
Research Technician, EnGGen

**Comment by alk224** [comment]
I believe I have fixed the issues raised here. The paper.md should be properly formatted and there is a bibtex file containing the references. I used the tool [here](http://www.snowelm.com/~t/doc/tips/makebib.en.html) to convert to bibtex and cleaned up the output manually. I also used [this tool](https://gist.github.com/arfon/478b2ed49e11f984d6fb) from the JOSS page to produce a metadata file (codemeta.json). Do you require that I also issue a new release that wraps all of these new files into it?

**Comment by arfon** [comment]
>  Do you require that I also issue a new release that wraps all of these new files into it?

Nope, we don't need those. Thanks for asking.

**Comment by arfon** [comment]
/ cc @openjournals/joss-reviewers - would anyone be willing to review this submission?

If you would like to review this submission then please comment on this thread so that others know you're doing a review (so as not to duplicate effort). Something as simple as `:hand: I am reviewing this` will suffice.

**Reviewer instructions**

- Please work through the checklist at the start of this issue.
- If you need any further guidance/clarification take a look at the reviewer guidelines here http://joss.theoj.org/about#reviewer_guidelines
- Please make a publication recommendation at the end of your review

Any questions, please ask for help by commenting on this issue! 🚀

**Comment by arfon** [comment]
@whedon commands

**Comment by tracykteal** [comment]
:hand: I am reviewing this. Thanks for your submission @alk224

**Comment by whedon** [comment]
Here are some things you can ask me to do:

```
# List all of Whedon's capabilities
@whedon commands

# Assign a GitHub user as the reviewer of this submission
@whedon assign @username as reviewer

# List the GitHub usernames of the JOSS editors
@whedon list editors

# List of JOSS reviewers together with programming language preferences and domain expertise
@whedon list reviewers

# Change editorial assignment
@whedon assign @username as editor

# Open the review issue
@whedon start review
```

:construction: **Important** :construction:

This is all quite new. Please make sure you check the top of the issue after running a @whedon command (you might also need to refresh the page to see the issue update).

**Comment by arfon** [comment]
@whedon editors

**Comment by tracykteal** [comment]
:hand: I am reviewing this. @alk224 thanks for your submission!

**Comment by arfon** [comment]
@whedon assign @arfon as editor

**Comment by whedon** [comment]
OK, the editor is @arfon

**Comment by arfon** [comment]
@whedon assign @tracykteal as reviewer

**Comment by whedon** [comment]
OK, the reviewer is @tracykteal

**Comment by arfon** [comment]
Friendly bump on this @tracykteal 😁

**Comment by alk224** [comment]
Someone just showed me a bug in the akutils test command. This happened as a result of a recent rewrite of the strip_primers command. The bug is fixed and the test command should function properly now.

**Comment by arfon** [comment]
> Friendly bump on this @tracykteal 😁

👋 @tracykteal - are you still able to complete this review for JOSS?

**Comment by arfon** [comment]
@alk224 - looks like @tracykteal isn't going to be able to do this review for us. Could you suggest any people from your research field who could potentially help with a review here?

**Comment by genomematt** [comment]
I will see if I can recruit a colleague

--=-=--=-=--=-=--=-=--=-=--
Matthew Wakefield
matthew@wakefield.id.au<mailto:matthew@wakefield.id.au>
+61 402 916 018

On 21 Jan 2017, at 4:30 am, Arfon Smith <notifications@github.com<mailto:notifications@github.com>> wrote:


@alk224<https://github.com/alk224> - looks like @tracykteal<https://github.com/tracykteal> isn't going to be able to do this review for us. Could you suggest any people from your research field who could potentially help with a review here?

-

**Comment by genomematt** [comment]
@tseemann<https://github.com/tseemann> @drpowell either of you interested (or appropriate delegation)

--=-=--=-=--=-=--=-=--=-=--
Matthew Wakefield
matthew@wakefield.id.au<mailto:matthew@wakefield.id.au>
+61 402 916 018

On 21 Jan 2017, at 4:30 am, Arfon Smith <notifications@github.com<mailto:notifications@github.com>> wrote:


@alk224<https://github.com/alk224> - looks like @tracykteal<https://github.com/tracykteal> isn't going to be able to do this review for us. Could you suggest any people from your research field who could potentially help with a review here?

-

**Comment by alk224** [comment]
@arfon @genomematt thanks for your help. I asked some colleagues about anyone they might suggest as well.

**Comment by alk224** [comment]
I haven't gotten any responses from willing individuals. To be clear for whomever does review this, you simply need a clean Ubuntu 14.04 OS installed natively or within a VM to run everything. Anyone new willing to take on this review? Please?

**Comment by drpowell** [comment]
@serine, you willing to review this?

**Comment by serine** [comment]
Thanks @drpowell, @alk224 yeh I'll be keen to do it. I've done a bit of work on S16 like profiling (viruses instead of bacteria) and used usearch and its related tools, but I also used vsearch faily extensively. My qiime knowledge are more theoretical then practical, never the less I've used most of the tools that qiime wraps around and will be happy to write it up. 

p.s @alk224 easy and clean install with miniconda, pip install also works, but user might need to install some tools manually.

p.s.s qiime2 is coming sometime soonish.. don't want to wait for it to come out? I think its mainly migration to python3

**Comment by arfon** [comment]
@whedon assign @serine as reviewer

**Comment by whedon** [comment]
OK, the reviewer is @serine

**Comment by arfon** [comment]
> Thanks @drpowell, @alk224 yeh I'll be keen to do it.

Wonderful, thanks @serine. Please take a look at the reviewer guidelines (http://joss.theoj.org/about#reviewer_guidelines) to get started. Basically, you just need to work through the check list above as you proceed. Feel free to ask questions here of the submitting authors.

**Comment by alk224** [comment]
@serine Thanks so much! 

miniconda install came after I started working on this. The installer here: https://github.com/alk224/akutils_ubuntu_installer as per the instructions here: http://alk224.github.io/akutils-v1.2/ will do all of the work for you.

QIIME2 will be released pretty soon, yes, but this is all based on 1.9/1.9.1 so if it doesn't get out soon it will be obsolete. The QIIME team have reached out about setting up an akutils front end for QIIME2, but I just don't have the time right now. I have several data sets that have been processed with this pipeline now so it would be nice to have something to reference directly.

**Comment by serine** [comment]
@arfon great, I'll get into it

**Comment by serine** [comment]
@alk224 Hey sorry took a bit of time to get to this and I also mis-understood what I was reviewing.. I got it :). Your lience missing year and holder 

```Copyright (c) <year> <copyright holders>``` bottom of the licence file

**Comment by serine** [comment]
@alk224 with your git repo. It's a little confusing (to me at least). Usually you'd have a single repo e.g `akutils` and `git clone http://blah/akults.git` and then you have branches and tags (releases) with different versions. I can see that you have at least one other akutils repo - [akutils-v1.1.1](https://github.com/alk224/akutils-v1.1.1). I'm sure you have a reason for that, but little confusing. I guess v1.2 is the next iteraction of v1.1 ?

**Comment by serine** [comment]
Am I missing something.. but can't get it going...

```
[bioinformatics-platform2]~/downloads/akutils-v1.2 ((v1.2.3))$ bash install 

Command autocompletion already configured for akutils.
	
Command PATH already configured for akutils.
	
akutils is properly configured. Restart your terminal or run:
source ~/.bashrc
	
[bioinformatics-platform2]~/downloads/akutils-v1.2 ((v1.2.3))$ akutils test
akutils: command not found
[bioinformatics-platform2]~/downloads/akutils-v1.2 ((v1.2.3))$ source ~/.bashrc
[bioinformatics-platform2]~/downloads/akutils-v1.2 ((v1.2.3))$ akutils test
akutils: command not found
```

I can get it going this way, but then I get stuck at the config step and it just hangs there

```
[bioinformatics-platform2]~/downloads/akutils-v1.2 ((v1.2.3))$ ./scripts/akutils test

No config file is present.  Running config utility to establish configurable
options.
	

This will help you configure your akutils config file for running akutils workflows.

First, would you like to configure your global settings or make a local config file
to override your global settings?  A local config file will reside within your
current directory.

Or else, you can choose rebuild if you want to generate a fresh global config file.
This is useful if you recently updated akutils and need to integrate newly available
configuration options.

Enter "global," "local," or "rebuild."

local

OK. Checking for existing config file in current directory.
(/home/kirill/downloads/akutils-v1.2/)
	

No config file detected in local directory. Creating new local config file.
```

**Comment by serine** [comment]
One other thing before I go to bed.. me personaly I don't like that you "silently" append stuff to `~/.bashrc`.  It's nice thought, but

**Comment by serine** [comment]
@arfon didn't meant to close this issue... my mistake, sorry

**Comment by alk224** [comment]
@serine Sorry about the confusion and the .bashrc. I can add in the license information.

Sometimes the source .bashrc doesn't seem to work and you just have to do a reboot. Also, I've had a few of the sources from the install script miss something this time or that and then just rerun the installer and it usually fills everything in. The .bashrc changes are meant to make it easy to use the akutils commands in a cluster environment where you can call all of the necessary modules rather than having the power to install stuff yourself. I also really enjoy autocomplete (referenced via .bashrc). v1.2 is much better than v1.1 which was my first major coding experience. It is rather clumsily written and lacks command integration and such. I transferred all of the files from v1.1 to 1.2 and rewrote most everything. I'm not good enough at git to really make use of branches, but I should have several releases of v1.2. It would make sense to run  
    akutils update  
after a reboot to ensure everything is working right. I just had to fix a bug due to a change in the biom command syntax which was breaking a few other commands.

**Comment by alk224** [comment]
@serine meant to use a code block there. Do:

    akutils update

**Comment by serine** [comment]
@alk224 sorry but I can't get it working. I took a quick pick at your `intall` script and I couldn't figure out what you are doing with the path.. if I simply `export PATH=/path/to/akutils-v1.2/scripts:$PATH` I can get `akutils` on cmd, but still the same problem - it finds no existing config file, then gives me options to select - I pick `local`.. and nothing happens. I think this is actually a bug. I took a pick at your `scripts/configure_slave.sh` line 120 has an `if` block with only `echo` nothing else happens in this block

**Comment by serine** [comment]
@arfon Is this intentional to have this question twice?

```
 A statement of need: Do the authors clearly state what problems the software is designed to solve and who the target audience is?
```

**Comment by alk224** [comment]
@serine Sorry about that. It is entirely likely there is a bug as all the sources seem to shift freely and without announcing that they will do so. I'll run a clean install on a VM in the morning and hopefully get this all working again.

**Comment by serine** [comment]
I think I missed something..Can akutils work without VM? or can it only work in a particular environmnet? Provided I have all of the tools installed it should work right?

**Comment by arfon** [comment]
> Its under Documentations and under Software paper

Yes, this is intentional. The point being that there should be a statement of need (presumably the same one?) in both the documentation and the paper.

**Comment by serine** [comment]
right, cool

**Comment by alk224** [comment]
@serine You want a clean Ubuntu 14.04 environment. See http://alk224.github.io/akutils-v1.2/

**Comment by serine** [comment]
What does that mean? Like you want no varibales set in your current shell? Can I just get a new shell and unset everything, something like this `env -i env`

**Comment by alk224** [comment]
@serine Generally just a fresh install using the 14.04.1 .iso. You could use a system that has other things installed, but given all the nuance of a QIIME install, you may find some things present or absent or incorrect versions of dependencies here and there. I have tried to enforce compatible versions during the install which could further break existing tools that you may like and want. I've never run `env -i env` so I'm not sure what that result might be. I know the installer will not work on Ubuntu 12.04 or 16.04 due to differences in package names in the respective repositories and probably similarly with the intermediate versions as well.

**Comment by serine** [comment]
To me first point is confusing 
> 1) Set up your system appropriately. I have done this on a large workstation as well as within a VirtualBox on my netbook. Install Ubuntu 14.04LTS. I have had trouble with 14.04.3 base installs so prefer 14.04.1 install and then update:
http://old-releases.ubuntu.com/releases/14.04.2/ubuntu-14.04.1-desktop-amd64.iso

I think you should either remove it or simplify to something like requires ubuntu 14.04.1 distro. It is still very restrictive. I would have thought that at least within ubuntu everything should be failry similar. Not sure if you want to go this far, but docker or other containers might be great - easy install solutions. 

For lets assume my env has been set appropriately and assume I have all dependencies installed

I've done 3 and 4. Now upto running `akutils` step.. and I'm having issues with configuration step..

would you be able to have a look in you `scripts/configure_slave.sh` line ~ 120 (I think in github its 123). There is an `if` block there with one statement `echo blah` and nothing else. I fall into that `if` block leading to `akutils` hanging until I `ctrl^C` ...

**Comment by alk224** [comment]
@serine I could specify 14.04.1, but that wouldn't strictly be true. My workstation at work is now on 14.04.5 because I continue to run all security updates. Ubuntu repos change with every release, and for some reason as they pack more bits into subsequent iterations of the same release (.1, .2 etc), the system behavior can change here and there. Annoying yes. I do not think I have the time to learn of something like docker given the complexity of the existing installer which I use to restore my workstations (I manage several, though I am a biologist by training) when someone messes them up or the system drive fails.

Curious on the config script. I checked that line of code and there is an open if statement, but it closes later. Look closely and you will see a series of nested conditionals which all close at lines 163, 169 and 170.

Perhaps there is an issue with the config script in that release (or did you clone the repo? That's what I would do). If you downloaded the release instead, try running a `git pull` from within the akutils directory and then try the akutils command with no arguments.

**Comment by serine** [comment]
My bad, yes `if` statement has bunch of other stuff in it, It was hard to read, cause of differnet indentations style.  

I followed installtion guide. I `git clone http:/blah/auktils-v1.2` I tested that, didn't work for me. I then `git checkout v1.2.3` which is your current release right? but still can't get pass configuration step

Any ideas what I am doing wrong?

**Comment by alk224** [comment]
That was one of my first exercises with conditionals. Some of my inexperience will still be glaring in terms of indents and such.

Can you delete your cloned akutils repo `rm -r ~/akutils-v1.2`

Then try a fresh clone of the repo in your $HOME directory followed by the bash install command from within the cloned directory?

**Comment by serine** [comment]
```
[bioinformatics-platform2]~$ pwd
/home/kirill
[bioinformatics-platform2]~$ echo $HOME
/home/kirill
[bioinformatics-platform2]~$ git clone https://github.com/alk224/akutils-v1.2.git
Cloning into 'akutils-v1.2'...
remote: Counting objects: 2870, done.
remote: Compressing objects: 100% (44/44), done.
remote: Total 2870 (delta 30), reused 0 (delta 0), pack-reused 2826
Receiving objects: 100% (2870/2870), 7.33 MiB | 2.94 MiB/s, done.
Resolving deltas: 100% (1805/1805), done.
Checking connectivity... done.
Checking out files: 100% (342/342), done.
[bioinformatics-platform2]~$ cd akutils-v1.2/
[bioinformatics-platform2]~/akutils-v1.2 (master)$ bash install 

Command autocompletion successfully configured for akutils.
		
Command PATH successfully configured for akutils.
		
akutils is properly configured. Restart your terminal or run:
source ~/.bashrc
	
[bioinformatics-platform2]~/akutils-v1.2 (master)$ at
atktopbm  atobm     
[bioinformatics-platform2]~/akutils-v1.2 (master)$ akutils test
akutils: command not found
[bioinformatics-platform2]~/akutils-v1.2 (master)$ export PATH=$HOME/akutils-v1.2/scripts:$PATH
[bioinformatics-platform2]~/akutils-v1.2 (master)$ akutils test

No config file is present.  Running config utility to establish configurable
options.
	

This will help you configure your akutils config file for running akutils workflows.

First, would you like to configure your global settings or make a local config file
to override your global settings?  A local config file will reside within your
current directory.

Or else, you can choose rebuild if you want to generate a fresh global config file.
This is useful if you recently updated akutils and need to integrate newly available
configuration options.

Enter "global," "local," or "rebuild."

local

OK. Checking for existing config file in current directory.
(/home/kirill/akutils-v1.2/)
	

No config file detected in local directory. Creating new local config file.
```

**Comment by alk224** [comment]
Does it hang right there?

**Comment by alk224** [comment]
And did you restart your terminal? I see you entered your own $PATH command.

**Comment by serine** [comment]
yep

**Comment by alk224** [comment]
OK, thanks for persisting. I will check it out in the morning at work on a clean VM.

**Comment by serine** [comment]
can you refresh, this github page. I removed mine `export` command. Yours worked fine after `source ~/.bashrc`

**Comment by alk224** [comment]
Wait it is working now?

**Comment by serine** [comment]
I can run `akutils` but it hangs at the configuration step :) 
```
[bioinformatics-platform2]~/akutils-v1.2 (master)$ pwd
/home/kirill/akutils-v1.2
[bioinformatics-platform2]~/akutils-v1.2 (master)$ which akutils 
/home/kirill/akutils-v1.2/scripts/akutils
[bioinformatics-platform2]~/akutils-v1.2 (master)$ akutils 

No config file is present.  Running config utility to establish configurable
options.
	

This will help you configure your akutils config file for running akutils workflows.

First, would you like to configure your global settings or make a local config file
to override your global settings?  A local config file will reside within your
current directory.

Or else, you can choose rebuild if you want to generate a fresh global config file.
This is useful if you recently updated akutils and need to integrate newly available
configuration options.

Enter "global," "local," or "rebuild."

local

OK. Checking for existing config file in current directory.
(/home/kirill/akutils-v1.2/)
	

No config file detected in local directory. Creating new local config file.
```

**Comment by alk224** [comment]
I see. That is less catastrophic than I was thinking. I'll fix this in the morning. I appreciate you bringing it to my attention.

**Comment by serine** [comment]
no probs, thanks

**Comment by alk224** [comment]
Starting on this now.

**Comment by alk224** [comment]
Looks like there are issues with pip and python. No surprise there in my experience as they change available module names pretty continuously. I know this all worked when I submitted akutils for review back in October. I will get it working again and then ping you on this thread to get the review going again. I don't think I will be able to completely solve these issues today or even this weekend. I really appreciate your willingness to review, and your patience with these issues. If you have used QIIME in the past, you might be accustomed to the rather arduous installs which is what my installer script is supposed to circumvent.

**Comment by serine** [comment]
Yeh no problem when you ready. 

Qiime has miniconda install now. I don't know how long they had it for, probably fairly recent, but it installs everything with ease. Might be worth while considering if you are having issues with pip install. From what I understand conda (miniconda) wraps everything into one "packages" such that there shouldn't be any issues with dependencies.  

In general it would be great if you could do few more stdout/stderr messages to keep user upto date with the progress. Like with that config option - I couldn't be sure if `akutils` was doing something or not. It might have been downloading and installing all the `pip` packages, but as far as I was conserned it was just handing there. Don't worry about this for now though.

p.s I'll be on annual next week, so might not hear from that week

**Comment by arfon** [comment]
How are we getting on here @alk224?

**Comment by arfon** [comment]
Friendly reminder on this @alk224

**Comment by arfon** [comment]
👋 @alk224, how are you getting on?

**Comment by alk224** [comment]
Hi Arfon, sorry for the delay. Working on some manuscripts right now so I
can graduate and plan to attend to akutils later this fall.

--
Lela Andrews
PhD Candidate, NAU Biological Sciences
Core Facility Manager, EnGGen

**Comment by arfon** [comment]
> Hi Arfon, sorry for the delay. Working on some manuscripts right now so I can graduate and plan to attend to akutils later this fall.

OK no problem. I'll remind you again sometime in early October if that works?

**Comment by arfon** [comment]
👋 @alk224 - are you ready to pick this up again?

**Comment by arfon** [comment]
If @alk224 doesn't re-appear in the next week I'll assume she's no longer interested in pursing this publication.

**Comment by alk224** [comment]
Hi @arfon. So sorry about the silence. I'm slowly reaching my goals. Problems with the installer we're identified last week, but until it is fully updated, I still can't address the reviewer comments with respect to silent modifications of .bashrc file. I'm still interested and will advise as soon as this is ready to move back toward review. Thanks for all your patience.

**Comment by arfon** [comment]
Thanks for the update @alk224. Just wanted to make sure you were still interested in pursuing this.

**Comment by alk224** [comment]
@arfon  thank you for hitting pause on this. Goals are being achieved (grad school completed, manuscripts prepared), but time remains short. Am hoping for time to really update this by December since by now Ubuntu 14.04 will see end of support in less than a year, and QIIME2 is now available with the wonderful provenance feature, but still lacks many options for performing a comprehensive ITS region analysis. As such, this should probably remain paused for now, and I will get back to this as soon as time allows, and simultaneously address reviewer concerns.

**Comment by arfon** [comment]
I've been in touch with @alk224 over email and she's agreed that at this point in time, it's best if she withdraws this submission.

@serine - many thanks for all of your help with this review, I'm sorry this paper didn't make it through to publication.

**Comment by serine** [comment]
all good, thanks to everyone and good luck
